src/idl/src/tree.c

```diff
@@ -772,6 +777,16 @@ uint32_t idl_bound(const void *node)
   return 0u;
 }
 
+uint16_t idl_bit_bound(const void *node)
+{
+  idl_mask_t mask = idl_mask(node);
+  if ((mask & IDL_BITMASK) == IDL_BITMASK)
+    return ((const idl_bitmask_t *)node)->bit_bound;
+  if ((mask & IDL_ENUM) == IDL_ENUM)
+    return ((const idl_enum_t *)node)->bit_bound;
+  return 0u;
+}
+
```

Comment:
We could opt to support this through `idl_is_bounded` and `idl_bound`. Not sure if that's desirable, but have you considered that possibility? There's of course the maximum of `64` that can be returned for a `@bit_bound`, but maybe it's pragmatic to repurpose those functions...

Reply:
Good point, I've merged the functions `idl_bit_bound` and `idl_bound`. I think `idl_is_bounded` for enum and bitmask types does not really make sense, as these types always have a bounded bit size.